.github/workflows/ci.yml

```
@@ -92,4 +88,109 @@ jobs:
+                  pull_number: context.issue.number,
+                  event: 'REQUEST_CHANGES',
+                  body: '🚫 **CI Build Failed**\n\nThe automated build process failed. Please review the build logs and fix the issues before requesting another review.\n\n**Next steps:**\n1. Check the build logs for specific errors\n2. Fix the identified issues\n3. Push your fixes to this branch\n4. The CI will automatically re-run\n\nOnce the build passes, this review will be dismissed automatically.'
+                });
+                console.log(`Auto-rejected PR #${context.issue.number} due to build failure`);
+              } catch (e) {
+                console.log(`Auto-reject review skipped/failed: ${e?.message ?? e}`);
+              }
+            } else {
+              console.log(`PR #${context.issue.number} already has a CI rejection review`);
+            }
+auto_approve_on_success:
```

actionlint (1.7.11)

[error] unexpected key "auto_approve_on_success" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)